.env

```diff
@@ -5,8 +5,8 @@ DEBUG=0
 USE_DOCKER=False
 DJANGO_SETTINGS_MODULE=config.settings.local
 DJANGO_SECRET_KEY=test-secret#-!key
-DATABASE_URL=psql://postgres:postgres@localhost:5432/postgres
-REDIS_URL=redis://localhost/0
-CELERY_BROKER_URL=redis://localhost/0
-ETHEREUM_NODE_URL=http://localhost:8545
-ETHEREUM_TRACING_NODE_URL=http://localhost:8545
+DATABASE_URL=psql://postgres:postgres@safe-transaction-service_db_1:5432/postgres
+REDIS_URL=redis://safe-transaction-service_redis_1/0
+CELERY_BROKER_URL=redis://safe-transaction-service_redis_1/0
+ETHEREUM_NODE_URL=http://host.docker.internal:8545
+ETHEREUM_TRACING_NODE_URL=http://host.docker.internal:8545
```

Comment:
The `host.docker.internal` is a special hostname that resolves to the Docker host ip address.